client/src/components/Round/Round.jsx

```diff
@@ -110,12 +121,16 @@ function Round() {
   if (error) return <Error error={error} />;
   const { round } = data;
 
+  if (forecastView && forecastViewDisabled(round)) {
+    setForecastView(false);
+  }
```

Comment:
I believe `set*` functions should not be called in the function body directly, but rather in response to some events. Here it may actually work because with the `if` makes sure it doesn't get called over and over, but I would do this instead:

```suggestion
  useEffect(() => {
    // Reset to default on round change
    setForecastView(false);
  }, [roundId]);
```

Reply:
Ah yeah the `if` was what made sure it didn't infinitely reload the page. I was looking for something proper like this so thanks for the suggestion. I moved it up next to the other `useEffect` call